src/main/java/javax/validation/Configuration.java

```diff
@@ -234,6 +249,20 @@
 	ParameterNameProvider getDefaultParameterNameProvider();
 
 	/**
+	 * Returns an implementation of the {@link ClockProvider}
+	 * interface following the default {@code ClockProvider}
+	 * defined in the specification:
+	 *
+	 * XXX BVAL-496 paste definition of the specification
```

Comment:
XXX is one of the convention for TODO that needs fixing before release. I used that consistently in the API/TCK/RI to mark all the places we need to update once the spec is written.